src/odemis/acq/stitching/test/tiledacq_test.py

```diff
@@ -0,0 +1,230 @@
+# -*- coding: utf-8 -*-
+from __future__ import division
+
+import logging
+import os
+import time
+import unittest
+from concurrent.futures._base import CancelledError, FINISHED
+
+import odemis
+import odemis.acq.stream as stream
+from odemis import model
+from odemis.acq.stitching._tiledacq import TiledAcquisitionTask, acquireOverviewMap
+from odemis.util import test
+from odemis.util.test import assert_pos_almost_equal
+
+logging.getLogger().setLevel(logging.DEBUG)
+
+CONFIG_PATH = os.path.dirname(odemis.__file__) + "/../../install/linux/usr/share/odemis/"
+SECOM_CONFIG = CONFIG_PATH + "sim/secom-focus-test.odm.yaml"
```

Comment:
Using this for now, though `cryosecom-sim` is using a different fake camera `andorcam2.AndorCam2` than the one tested in here `simcam.Camera`